<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRTableColumnValuesDialog</class>
 <widget class="QDialog" name="QFRDRTableColumnValuesDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>first value:</string>
       </property>
       <property name="buddy">
        <cstring>edtStart</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFDoubleEdit" name="edtStart"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>last value:</string>
       </property>
       <property name="buddy">
        <cstring>edtEnd</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFDoubleEdit" name="edtEnd"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>type:</string>
       </property>
       <property name="buddy">
        <cstring>cmbType</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbType">
       <item>
        <property name="text">
         <string>linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>logarithmic</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>value count:</string>
       </property>
       <property name="buddy">
        <cstring>spinCount</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>preview:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labPreview">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>insert as ...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="radColumn">
       <property name="text">
        <string>column</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="radRow">
       <property name="text">
        <string>row</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
   <slots>
    <signal>valueChanged(double)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFRDRTableColumnValuesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFRDRTableColumnValuesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtStart</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFRDRTableColumnValuesDialog</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtEnd</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFRDRTableColumnValuesDialog</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>QFRDRTableColumnValuesDialog</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QFRDRTableColumnValuesDialog</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePreview()</slot>
 </slots>
</ui>
